specs/0119-transactional-emails.md

```diff
@@ -0,0 +1,428 @@
+
+Each localization community has its own specific workflows, so do reach out to your [Team Managers](https://pontoon.mozilla.org/*/contributors/) to learn more. 
+
+**Team roles**  
+Speaking of team managers, teams can have up to 3 different roles. While all roles contribute translations for their language, some roles have additional responsibilities:
+
+1. Team Manager - Team managers are responsible for managing and growing their language community, including: mentoring new contributors, providing feedback on their translation submissions, and creating language resources. Each locale will have at least one team manager.   
+2. Translator - Translators are responsible for reviewing translation submissions and providing feedback to team contributors. Community members that have been evaluated by the team manager and other translators can be assigned the translator role.  
+3. Contributor - Contributors submit translation suggestions in Pontoon for their target language. New members of a language community start as a contributor.   
+   
+
+We’ve found that the best way for new contributors to learn and grow within their community is to network with other team members and proactively request feedback from translators and managers on their submissions. If you have trouble reaching other team members or getting feedback, let us or the project manager know (see below).
+
+**Project manager**  
+Finally, there is one key point of contact to support you in your translation journey — the project manager. Each project on Pontoon has a project manager assigned to act as the primary point of contact for the project itself. They are there to answer your questions and assist with any problems you are having related to translation of project strings.
+
+If you have a question or issue related to a particular string, you can click the “REQUEST CONTEXT or REPORT ISSUE” button from the translation view in Pontoon. That will create a comment you can fill in with the details and it will notify the project manager. You can also find the project manager’s contact information in their profile by clicking on their name in the field at the top of the project page.
+
+We hope this gives you a better understanding of how localization works at Mozilla. 
+
+If you have any questions, reach out any time to the Mozilla localization team on [Matrix chat](https://matrix.to/#/#l10n-community:mozilla.org) or [Discourse](https://discourse.mozilla.org/c/l10n/547).
+
+Thanks,  
+Pontoon Team
+
+#### Frequency
+
+Once
+
+### Onboarding: Week 1 - more localization resources
+
+#### Trigger
+
+When both conditions are met: 
+
+1. At least 1 week after account creation  
+2. 5 or more translation suggestions submitted
```

Comment:
I assume we should only check these two conditions once? Otherwise we could end up sending this "final introductory email" years after the account creation.

Reply:
OK, so essentially we should change line 236 to "When either condition is met: "?

Reply:
Summarizing our discussion offline: with my initial conditions there's a cost associated with needing to check every user that registers to see if they have made 5 translation suggestion. 

Reviewing the spec again I'd like to suggest some changes - I'd like to change the number of translations from 5 to 1. (The reasoning behind this is that the Inactive account: Contributor email triggers with just 1 translation). Then for how to handle the sending so its at least 1 week away from account creation:

I'd like to propose two alternatives:
Alternative 1:
When user makes their first translation, trigger an email to be sent 1 week from date of trigger.

Alternative 2:
Check once a week, starting 1 week from account creation date until 6 months from account creation date. (We consider an account inactive at 6 months, so we can assume they are unlikely to be a contributor at this point.)

I'm fine with either approach, but would like your input @mathjazz  on which is more realistic from a technical standpoint.

Reply:
Okay, let's just go with the simple implementation. I'll adjust the text accordingly.